frontend/src/store/modules/visitors/actions.js

```diff
@@ -34,6 +35,9 @@ import {getTimeByPeriod} from '@/services/periodService';
 export default {
     [CHANGE_SELECTED_PERIOD]: (context, payload) => {
         context.commit(SET_SELECTED_PERIOD, payload.value);
+        context.commit(FETCH_TABLE_DATA, context.state.tableData.groupedParameter);
+        context.commit(FETCH_LINE_CHART_DATA, context.state.tableData.groupedParameter);
+        context.commit(GET_BUTTON_DATA, context.state.activeButton);
```

Comment:
Action is called "FETCH_BUTTON_DATA", please, check. Also, When a period is changed you should update all values on buttons. You can add action FETCH_EACH_BUTTON_DATA: and fetchin data for each button from buttonTypes config